### PR TITLE
Allow passing custom placements config

### PR DESCRIPTION
**Pull request by evgenykochetkov**
Addresses one of the problems described in #108

### CODE REVIEW

**Comment by yesmeck** [comment]
@benjycui  帮忙 review  下

cc @valleykid

File: README.md [hunk 1]
```diff
@@ -186,6 +186,12 @@ ReactDOM.render(<Menu>
             <th>() => document.body</th>
             <td>Where to render the DOM node of popup menu when the mode is horizontal or vertical</td>
         </tr>
+        <tr>
+            <td>popupPlacements</td>
```
Comment (benjycui) [inline README.md]: How about `builtinPlacements`? To be consistent with [`rc-trigger`](https://github.com/react-component/trigger#props)

**Comment by benjycui** [comment]
Code is OK, just update the API name.

File: README.md (hunk 1 above)
Comment (evgenykochetkov) [reply README.md]: Done 👍🏻

**Comment by valleykid** [comment]
Thanks for your PR, I'll merge asap.

File: README.md [hunk 2]
```diff
@@ -186,6 +186,12 @@ ReactDOM.render(<Menu>
             <th>() => document.body</th>
             <td>Where to render the DOM node of popup menu when the mode is horizontal or vertical</td>
         </tr>
+        <tr>
+            <td>builtinPlacements</td>
```
Comment (yesmeck) [inline README.md]: Should rename to `placements`.

Comment (yesmeck) [inline README.md]: Should rename to `placements`.

**Comment by afc163** [comment]
Conflict

**Comment by picodoth** [comment]
@evgenykochetkov will you be able to do a rebase? Let's close this one soon if possible, thanks!

**Comment by borisrorsvort** [comment]
@picodoth Would be nice to update the examples page with an example similar to this one https://github.com/react-component/menu/issues/122#issuecomment-389991276

**Comment by picodoth** [comment]
seeing https://github.com/react-component/menu/pull/156

**Pull request closed without merging**